pkg/webui/components/radio-button/radio-button.styl

```diff
@@ -12,58 +12,81 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-.container
-  display: flex
+.wrapper
+  cursor: pointer
```

Comment:
When in a horizontal group, the wrapper needs more spacing to the right.
![image](https://user-images.githubusercontent.com/5710611/59428189-e4a74600-8ddc-11e9-9fe7-ead9ca948238.png)